docs/topics/install.txt

```diff
@@ -150,7 +150,7 @@ This is the recommended way to install Django.
         $ python -m pip install Django
 
 .. _pip: https://pip.pypa.io/
-.. _standalone pip installer: https://pip.pypa.io/en/latest/installing/#installing-with-get-pip-py
+.. _standalone pip installer: https://pip.pypa.io/en/latest/installation/
```

Comment:
Verified, old URL was 'this page has moved' with a soft redirect (`<meta>` or JS) to the new URL given.